admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -1,65 +1,79 @@
+``cron.php`` is an ownCloud internal process that runs such background jobs on demand.
 
-Parameters
-----------
-In the admin settings menu you can configure how cron-jobs should be executed.
-You can choose between the following options:
-
--   AJAX
--   Webcron
--   Cron
+ownCloud plug-in applications can register actions with ``cron.php`` automatically to take care of typical housekeeping operations. 
+These actions can include garbage collecting of temporary files or checking for newly updated files using ``filescan()`` on externally mounted file systems.
 
 Cron Jobs
 ---------
 
-You can schedule cron jobs in three ways -- using AJAX, Webcron, or cron. The default method is to use AJAX.  However, the recommended method is to use cron.  The following sections describe the differences between each method.
+You can schedule Cron jobs in three ways: `AJAX`_, `Webcron`_, or `Cron`_. 
+These can all be configured in the admin settings menu. 
+The default method is to use AJAX.  
+However, the recommended method is to use Cron.  
+The following sections describe the differences between each method.
 
 AJAX
 ~~~~
 
-The AJAX scheduling method is the default option.  Unfortunately, however, it is also the least reliable. Each time a user visits the ownCloud page, a single background job is executed. The advantage of this mechanism is that it does not require access to the system nor registration with a third party service. The disadvantage of this mechanism, when compared to the Webcron service, is that it requires regular visits to the page for it to be triggered.
+The AJAX scheduling method is the default option. 
+However, it is also the *least* reliable. 
+Each time a user visits the ownCloud page, a single background job is executed. 
+The advantage of this mechanism, however, is that it does not require access to the system nor registration with a third party service. 
+The disadvantage of this mechanism, when compared to the `Webcron`_ service, is that it requires regular visits to the page for it to be triggered.
 
 .. note:: Especially when using the Activity App or external storages, where new
-   files are added, updated or deleted one of the two methods below should be
+   files are added, updated, or deleted one of the two methods below should be
    preferred.
 
 Webcron
 ~~~~~~~
 
-By registering your ownCloud ``cron.php`` script address at an external webcron
-service (for example, easyCron_), you ensure that background jobs are executed
-regularly. To use this type of service, your server you must be able to access
-your server using the Internet. For example::
+By registering your ownCloud ``cron.php`` script address as an external webcron service (for example, easyCron_), you ensure that background jobs are executed regularly. 
+To use this type of service, your server you must be able to access your server using the Internet. For example::
```

Comment:
Sugested rewording:
To use this type of service, your external webcron service must be able to access your ownCloud server using the Internet.